test/contrib/test_socks.py

```diff
@@ -199,6 +206,8 @@ class TestSocks5Proxy(IPV4SocketDummyServerTestCase):
     Test the SOCKS proxy in SOCKS5 mode.
     """
     def test_basic_request(self):
+        disable_warnings(NoIPv6Warning)
```

Comment:
As a follow-on question: can we just remove this warning entirely? What's the point of having it and always turning it off, given that we raise it from our own test infrastructure anyway.